src/travel-details-screens/components/TripSection.tsx

```diff
@@ -177,11 +231,45 @@ export const TripSection: React.FC<TripSectionProps> = ({
             <SituationMessageBox noStatusIcon={true} situation={situation} />
           </TripRow>
         ))}
-        {notices.map((notice) => (
-          <TripRow rowLabel={<ThemeIcon svg={Info} />}>
+        {notices.map((notice, i) => (
+          <TripRow key={i} rowLabel={<ThemeIcon svg={Info} />}>
```

Comment:
Here using index as key might actually introduce bugs. Also using index is the default fallback that React uses, so adding it makes no difference other than removing the warning. Does the notice have an id that can be used?